### PR TITLE
92 erik paper branch

### PR DESCRIPTION
**Pull request by eriktamsen**
new update of paper branch with improvements to workflow

### CODE REVIEW

File: usecases/optimization_paper/tex/tex_sections/appendix_fem_model.tex [hunk 1]
```diff
@@ -108,37 +108,40 @@ \subsubsection{Macroscopic tangent}
...
-Both compressive and tensile strength can be approximated using an generalized exponential function,
+\subsection{Coupling Material Properties to Degree of Hydration} \label{appendix:fem_evolution}
+\subsubsection{Compressive strength}
+The compressive strength in terms of the degree of hydration can be approximated using an exponential function, c.f. \cite{car_2016_mamt},
 \begin{align}
-	\strengthX(\DOH) = \DOH(\zeit)^{\strengthXExp} \strengthXInf. \label{eq:mechanics-hydration}
+	\fc(\DOH) = \DOH(\zeit)^{\strengthCExp} \fcInf \label{eq:fcwrtDOH}
 \end{align}
-This model has two parameter, $\strengthXInf$, the value of the parameter at full hydration, $\DOH = 1$ and $\strengthXExp$ the exponent, which is a purely numerical parameter, difficult to estimate directly from a mix design, as the mechanisms are quite complex.
+This model has two parameter, $\fcInf$, the compressive strength of the parameter at full hydration, $\DOH = 1$ and $\strengthCExp$ the exponent, which is a purely numerical parameter.
```
Comment (srosenbu) [inline usecases/optimization_paper/tex/tex_sections/appendix_fem_model.tex:117]:
```suggestion
This model has two parameters, $\fcInf$, the compressive strength of the parameter at full hydration, $\DOH = 1$ and $\strengthCExp$ the exponent, which is a purely numerical parameter.

```

File: usecases/optimization_paper/tex/tex_sections/concrete_model.tex [hunk 2]
```diff
@@ -1,43 +1,39 @@
...
-
-
-....
-Figure xy shows the time evaolution of DOH, E and fc for the set of paramters given in Table XY, with some variations relevant to the respective models, as presented in the figure..
-
-
-
+For a detailed description of the parameters evolution with respect to the degree of hydration see in Appendix \ref{appendix:fem_evolution}.
+Figure \ref{fig:parameterEvolution} shows the influence of the different parameters.
+In addition to the presented formulations in \cite{car_2016_mamt} which depend a theoretical value of parameters for fully hydrated concrete at $\DOH = 1$, this work reformulates the equations, to depend on the 28 day values $\eModTwentyEight$ and $\fcTwentyEight$ as well as the corresponding $\DOHTwentyEight$ which is obtained via a simulation.
```
Comment (srosenbu) [inline usecases/optimization_paper/tex/tex_sections/concrete_model.tex:34]:
```suggestion
In addition to the presented formulations in \cite{car_2016_mamt} which depend on a theoretical value of parameters for fully hydrated concrete at $\DOH = 1$, this work reformulates the equations, to depend on the 28 day values $\eModTwentyEight$ and $\fcTwentyEight$ as well as the corresponding $\DOHTwentyEight$ which is obtained via a simulation.

```

File: usecases/optimization_paper/tex/tex_sections/introduction.tex [hunk 3]
```diff
@@ -2,52 +2,47 @@
 The controlled production environment allows for higher quality products and enables the mass production of elements.
 In the standard design approach, engineers or architects select a structure, estimate the loads, choose mechanical properties, and design the element accordingly. 
 If the results are not satisfactory, the required mechanical properties are iteratively adjusted, aiming to improve the design.
-This approach is fine, when the choice of mixtures limited and the expected concrete properties are well known.
+This approach is fine, when the choice of mixtures is limited and the expected concrete properties are well known.
 There are various published methods to automate this process and optimize the beam design at this level.
-Computer aided beam design optimization dates back  at least 50 year, e.g. \cite{Haung1967}.
-Generally the objective is reducing costs, with the design variables being the beam geometry, the amount and location of the reinforcement and sometimes the compressive strength of the concrete.
+Computer aided beam design optimization dates back at least 50 year, e.g. \cite{Haung1967}.
```
Comment (srosenbu) [inline usecases/optimization_paper/tex/tex_sections/introduction.tex:7]:
```suggestion
Computer aided beam design optimization dates back at least 50 years, e.g. \cite{Haung1967}.

```

**Review by srosenbu — APPROVED** [review]
Looks good. I just found some spelling mistakes ;)

**Review by joergfunger — COMMENTED** [review]
I will send corrections on the paper itself separately. Should we delete the branch?

**Pull request merged**